lib/shared/screens/admin/shared/components/options-list/option.jsx

```diff
@@ -101,15 +102,18 @@ export default class Option extends Component {
       OptionsList,
       value,
       white,
+      type,
```

Comment:
yea i did that before but the warning that value was passed as a string instead of a boolean came up. 

should i change the proptype to string and then cast to boolean?